mods/fire/init.lua

```diff
@@ -249,21 +259,16 @@ else
 		end,
 	})
 
-	-- Remove basic flames and flammable nodes
+	-- Remove flammable nodes
 
 	minetest.register_abm({
 		nodenames = {"fire:basic_flame"},
```

Comment:
well, adding the `neigbors` is indeed great, but I'm a bit hesitant to remove the `if p then` part, since, while it would be an extreme edge case (something in C++ breaking badly somehow when find_node_near called), it would crash hard if it were `nil`.